salt/modules/file.py

```diff
@@ -1411,45 +1411,62 @@ def line(path, content, match=None, mode=None, location=None,
     '''
     .. versionadded:: 2015.8.0
 
-    Edit a line in the configuration file.
+    Edit a line in the configuration file. The ``path`` and ``content``
+    arguments are required, as well as passing in one of the ``mode``
+    options.
 
-    :param path:
+    .. note::
```

Comment:
Yeah that probably makes sense. I wasn't sure if that should have been exposed up higher in the docs or not. I'll move it. 👍